docs/compose-dynamic-conditions.md

```diff
@@ -0,0 +1,21 @@
+# Compose Dynamic Conditions
```

Comment:
I think it would be better to rewrite the article
The structure of the article:
What dooes it mean **Compose Dynamic Conditions** ?
Why we should use  **Compose Dynamic Conditions** ?
When we should use  **Compose Dynamic Conditions** ?
Description how **Compose Dynamic Conditions** implemented
Example